profiles/arch/arm/use.mask

```diff
@@ -5,6 +5,13 @@
 -cpu_flags_arm_iwmmxt
 -iwmmxt
 
+# Jonathan Scruggs <j.scruggs@gmail.com> (10 Aug 2017)
```

Comment:
I honestly doubt you're going to get all those flags global, so they belong in `package.use`.

Reply:
Oh good catch. I opened use.mask instead of package. Oops. I did mean package mask. :)

Reply:
D'oh. Did it for all three.